homeassistant/components/ezviz/camera.py

```diff
@@ -210,20 +231,22 @@ async def async_setup_entry(hass, entry, async_add_entities):
     )
 
 
-class EzvizCamera(CoordinatorEntity, Camera, RestoreEntity):
+class EzvizCamera(CoordinatorEntity, Camera):
     """An implementation of a Ezviz security camera."""
 
+    coordinator: EzvizDataUpdateCoordinator
+
     def __init__(
         self,
-        hass,
-        coordinator,
-        idx,
-        camera_username,
-        camera_password,
-        camera_rtsp_stream,
-        local_rtsp_port,
-        ffmpeg_arguments,
-    ):
+        hass: HomeAssistant,
```

Comment:
Side note: We could pass the ffmpeg data instead of `hass` since we only use the former.